### PR TITLE
(dev/core#4485) Replace Event Total by Total Amount

### PR DESCRIPTION
**Pull request by yashodha**
Overview
----------------------------------------
Replace _Event Total_ by Total Amount

Before
----------------------------------------
Event Total doesn't sound right.

After
----------------------------------------
Keep the verbiage standardized.

### CODE REVIEW

File: templates/CRM/Price/Form/LineItem.tpl [hunk 1]
```diff
@@ -108,7 +108,7 @@
     {if $context EQ "Contribution"}
       {ts}Contribution Total{/ts}:
     {elseif $context EQ "Event"}
-      {ts}Event Total{/ts}:
+      {ts}Total Amount{/ts}:
```
Comment (larssandergreen) [inline templates/CRM/Price/Form/LineItem.tpl:111]: Could just let this use the else below

**Comment by larssandergreen** [comment]
This makes sense to me.

**Comment by larssandergreen** [comment]
Actually, thinking about this for a minute, why do we say Total Amount instead of just Total, like every receipt ever does?

**Comment by mlutfy** [comment]
> "Actually, thinking about this for a minute, why do we say Total Amount instead of just Total, like every receipt ever does?"

It's just under "Tax Total", so it would normally make sense, except that CiviCRM doesn't display- multiple taxes, or rarely, and even if it did, there are rarely more than two taxes, so nobody cares about a tax total, just a total for the payment.

We tend to do stuff like this, using the taxcalculator ext:

image

So I would also strongly support:

- Rename "Tax Total" to  "Taxes"
- Rename "Total Amount" to "Total"

**Comment by yashodha** [comment]
@mlutfy 

Should we merge this PR? 
And have another PR for the following if we have consensus since these will affect Contribution/Membership/Event registration (a lot more occurrences for _Total Amount_)
-     Rename "Tax Total" to "Taxes"
-     Rename "Total Amount" to "Total"

We also have other strings like _Contribution Total_, etc that need renaming as well. We might to take this  opportunity to look at all other strings as well. Thanks!

**Comment by mlutfy** [comment]
@yashodha yep ok, I guess the renaming is a larger conversation, and this is a quickfix. Thanks!

**Pull request merged**